parsing/dialects/bigquery/parserImpls.ftl

```diff
@@ -14,3 +14,23 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 -->
+
+SqlNode ExceptExpression(List<SqlNode> selectList):
+{
+    final Pair<SqlNodeList, SqlNodeList> nameAndTypePair;
+    final SqlNodeList exceptList;
+}
+{
+    <EXCEPT> nameAndTypePair = ParenthesizedCompoundIdentifierList()
+    {
+        exceptList = nameAndTypePair.getKey();
+        if (selectList.size() > 0) {
+            SqlIdentifier identifier = (SqlIdentifier) selectList.get(0);
+            if (!identifier.toString().equals("*") || selectList.size() != 1) {
```

Comment:
Is it possible to have more than 1 column in the selectList and still be valid? For example, what about:
SELECT *, CURRENT_DATE as date EXCEPT(foo.a) FROM foo

Reply:
I don't think so but all I have to go off of is the BigQuery docs which only have SELECT * EXCEPT(foo) as a valid option. What do you think @dasch-1 ?

Reply:
Based on my reading of the BigQuery docs, only SELECT(*) EXCEPT(...) is valid.